CMakeLists.txt

```diff
@@ -276,15 +277,15 @@ if(CMAKE_SYSTEM_NAME STREQUAL "OS400")
 endif()
 
 if(SENTRY_TRANSPORT_CURL)
-	if(NOT CURL_FOUND) # Some other lib might bring libcurl already
+	if(NOT TARGET CURL::libcurl) # Some other lib might bring libcurl already
 		find_package(CURL REQUIRED COMPONENTS AsynchDNS)
 	endif()
 
 	target_link_libraries(sentry PRIVATE CURL::libcurl)
 endif()
 
 if(SENTRY_TRANSPORT_COMPRESSION)
-	if(NOT ZLIB_FOUND)
+	if(NOT TARGET ZLIB::ZLIB)
 		find_package(ZLIB REQUIRED)
```

Comment:
There is no problem with calling `find_package` repeatedly. However, you are really interested in `ZLIB::ZLIB` already being created, e.g. by a subproject.